src/main/java/com/owncloud/android/lib/common/operations/RemoteOperation.java

```diff
@@ -120,6 +131,38 @@ public RemoteOperationResult execute(Account account, Context context) {
         return run(mClient);
     }
 
+    /**
+     * Synchronously executes the remote operation on the received ownCloud account.
+     * <p>
+     * Do not call this method from the main thread.
+     * <p>
+     * This method should be used whenever an ownCloud account is available, instead of
+     * {@link #execute(NextcloudClient)}.
+     *
+     * @param account ownCloud account in remote ownCloud server to reach during the
+     *                execution of the operation.
+     * @param context Android context for the component calling the method.
+     * @return Result of the operation.
+     */
+    public RemoteOperationResult executeNextcloudClient(Account account, Context context) {
```

Comment:
should the parmters be annotated with `@NonNull´ ?

Reply:
I just did a c&p, but this is indeed a good idea. So we make sure that calling class is already handling possible null correctly. :+1: